targets/lenovo-x1-carbon.nix

```diff
@@ -268,6 +268,13 @@
             environment.etc.${getAuthKeysFilePathInEtc} = getAuthKeysSource {inherit pkgs;};
             services.openssh = sshAuthorizedKeysCommand;
 
+            # Enable TPM2 PKCS11 interface
+            security.tpm2.enable = true;
+            security.tpm2.pkcs11.enable = true;
+            security.tpm2.tctiEnvironment.enable = true;
+            environment.systemPackages = with pkgs; [tpm2-tools opensc];
+            users.users.ghaf.extraGroups = ["tss"];
```

Comment:
Place this under users and conditional on TPM being enabled or not., as we are trying to centralize users and groups so. to keep it aligned.